src/Localization/en/GraphAPICompute.json

```diff
@@ -0,0 +1,58 @@
+  "CosmosD16s": "Cosmos.D16s (General Purpose Cosmos Compute with 16 vCPUs, 64 GB Memory)",
+  "CosmosD32s": "Cosmos.D32s (General Purpose Cosmos Compute with 32 vCPUs, 128 GB Memory)",
+  "CreateMessage": "Graph GraphAPI Compute resource is being created.",
+  "CreateInitializeTitle": "Provisioning resource",
+  "CreateInitializeMessage": "GraphAPI Compute resource will be provisioned.",
+  "CreateSuccessTitle": "Resource provisioned",
+  "CreateSuccesseMessage": "GraphAPI Compute resource provisioned.",
+  "CreateFailureTitle": "Failed to provision resource",
+  "CreateFailureMessage": "GraphAPI Compute resource provisioning failed.",
+  "UpdateMessage": "GraphAPI Compute resource is being updated.",
+  "UpdateInitializeTitle": "Updating resource",
+  "UpdateInitializeMessage": "GraphAPI Compute resource will be updated.",
+  "UpdateSuccessTitle": "Resource updated",
+  "UpdateSuccesseMessage": "GraphAPI Compute resource updated.",
+  "UpdateFailureTitle": "Failed to update resource",
+  "UpdateFailureMessage": "GraphAPI Compute resource updation failed.",
+  "DeleteMessage": "GraphAPI Compute resource is being deleted.",
+  "DeleteInitializeTitle": "Deleting resource",
+  "DeleteInitializeMessage": "GraphAPI Compute resource will be deleted.",
+  "DeleteSuccessTitle": "Resource deleted",
+  "DeleteSuccesseMessage": "GraphAPI Compute resource deleted.",
+  "DeleteFailureTitle": "Failed to delete resource",
+  "DeleteFailureMessage": "GraphAPI Compute resource deletion failed.",
+  "CannotSave": "Cannot save the changes to the GraphAPI Compute resource at the moment.",
+  "GraphAccountEndpoint": "Graph Account Endpoint",
+  "NoValue": "",
+  "CosmosD4Details": "General Purpose Cosmos Compute with 4 vCPUs, 16 GB Memory",
+  "CosmosD8Details": "General Purpose Cosmos Compute with 8 vCPUs, 32 GB Memory",
+  "CosmosD16Details": "General Purpose Cosmos Compute with 16 vCPUs, 64 GB Memory",
+  "ApproximateCost": "Approximate Cost Per Hour",
+  "CostText": "Hourly cost of the GraphAPI Compute resource depends on the SKU selection, number of instances per region, and number of regions.",
+  "ConnectionString": "Connection String",
+  "ConnectionStringText": "To use the GraphAPI Compute, use the connection string shown in ",
+  "KeysBlade": "the keys blade.",
+  "MetricsString": "Metrics",
+  "MetricsText": "Monitor the CPU and memory usage for the GraphAPI Compute instances in ",
+  "MetricsBlade": "the metrics blade.",
+  "MonitorUsage": "Monitor Usage",
+  "ResizingDecisionText": "Number of instances has to be 1 during provisioning. Instances can only be incremented by 1 at once. ",
+  "ResizingDecisionLink": "Learn more about GraphAPI Compute sizing.",
+  "WarningBannerOnUpdate": "Adding or modifying GraphAPI Compute instances may affect your bill.",
+  "WarningBannerOnDelete": "After deprovisioning the GraphAPI Compute, you will not be able to connect to the Graph API account."
```

Comment:
>deprovisioning [](http://example.com/codeflow?start=34&length=14)

in that case - do we even want to show deprovision?

Reply:
The data (graphs) is not deleted. There is no way to change instance size without deleting and recreating. 
To be honest, there are conversations around removing the support to deprovision but the above reason keeps us from it. For now, providing  deprovisioning support is the way to go. In case it changes, I would come back to reflect the change here.

Reply:
I thought Oliver was mentioning that this compute unlike sqlx needs to be opted in or during accoutn creation. so if we deprovision, will another reprovisiong on the existing db work?